ledger_analytics/development.py

```diff
@@ -0,0 +1,418 @@
+            include_process_risk: should process risk or
+                aleatoric uncertainty be included in the predictions.
+                Defaults to ``True``. If ``False``, predictions are
+                based on the mean function, only.
+        """
+
+        include_process_risk: bool = True
+
+
+class ManualATA(DevelopmentModel):
+    """ManualAta.
+
+    This model is different from other loss development models in that it uses
+    hard-coded age-to-age factors, rather than estimating them from the data.
+
+    ..  math::
+      
+        y_{ij} &= \mu_{ij}\\\\
+        \mu_{ij} &= ATA_{j - 1} y_{ij-1}\\\\
+        \\bf{ATA} &= \\text{user input}
+
+    The primary intended use case of this model is for supporting workflows where
+    age-to-age factors are selected or adjusted by hand, or age-to-age factors
+    are provided by a bureau or other external source without any supporting
+    data behind them.
+
+    See the model-specific documentation in the User Guide for more details.
+
+    The fit and predict configurations are controlled by :class:`Config` and
+    :class:`PredictConfig` classes, respectively.
+    """
+
+    class DefaultPriors(Enum):
+        pass
+
+    class Config(ValidationConfig):
+        """ManualAta model configuration class.
+
+        Attributes:
+            loss_definition: the field to model in the triangle. One of
+                ``"paid"`` ``"reported"`` or ``"incurred"``.
+            ata_factors: a list of age-to-age development factors. If
+                ``development_resolution`` is 3 and ``development_offset`` is 6, then the
+                first factor in the list should be a 6-to-9 month ATA, the second factor
+                should be a 9-to-12 month ATA, and so forth.
+            development_resolution: the number of months between development lags
+                of successive development factors.
+            development_offset: the development lag (in months) of the first
+                development factor provided.
+        """
+
+        loss_definition: Literal["paid", "reported", "incurred"] = "paid"
+        ata_factors: list[float]
+        development_resolution: int = 12
+        development_offset: int = 0
+
+    class PredictConfig(ValidationConfig):
+        """TraditionalChainLadder predict configuration class.
+
+        Attributes:
+            max_dev_lag: the maximum development lag to predict to.
+        """
+
+        max_dev_lag: int | None = None
+
+
+class MeyersCRC(DevelopmentModel):
+    """Bayesian MeyersCRC.
+
+    This model implements the MeyersCRC model from 
+    `Glenn Meyer's 2019 monograph <https://www.casact.org/sites/default/files/2021-02/08-Meyers.pdf>`_:
+
+        * Meyers (2019). Stochastic loss reserving using Bayesian MCMC models (2nd edition).
+            Casualty Actuarial Society.
+
+    The model has the form:
+
+    ..  math::
+
+        y_{ij} &\sim \mathrm{Gamma}(\mu_{ij}, \\sigma_{ij}^2) \quad{\\forall j \in (1, M]}\\\\
```

Comment:
The CRC is technically on the loss ratio scale in our implementation, should we make that clear here too? e.g., 
```suggestion
\mathrm{LR}_{ij} &\sim \mathrm{Gamma}(\mu_{ij}, \\sigma_{ij}^2) \quad{\\forall j \in (1, M]}\\\\
```

Reply:
Ah nevermind I see the "where y represents loss ratio" below now!